web/server/codechecker_server/server.py

```diff
@@ -84,6 +83,24 @@ def log_message(self, msg_format, *args):
         """ Silencing http server. """
         return
 
+    def send_exception(self, error_msg, iprot, oprot, otrans):
```

Comment:
Please extend the function name and the documentation to reflect that this method sends exceptions converted to a valid thrift exception.